src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -22,10 +23,17 @@ class ResultView {
         }
 
         fun printWinningStatistics(winningStatistics: WinningStatistics) {
-            (3..6).forEach {
+            listOf(
+                Rank.FIFTH,
+                Rank.FOURTH,
+                Rank.THIRD,
+                Rank.SECOND,
+                Rank.FIRST,
+            ).forEach {
                 val prize = LottoCalculator.calculatePrizeMoney(it)
```

Comment:
아래와 같이 rank의 프로퍼티인 `winningMoney` 값을 getter로 호출하면 되지 않을까요? 🤔

```suggestion
                val prize = it.winningMoney
```

Reply:
이렇게 고치니 LottoCalculator.calculatePrizeMoney가 필요가 없군요. 모두 삭제 하도록 하겠습니다.